boards/adafruit-itsybitsy-m4/doc.txt

```diff
@@ -0,0 +1,37 @@
+/**
+@defgroup    boards_adafruit-itsybitsy-m4 Adafruit-Itsybitsy-M4
+@ingroup     boards
+@brief       Support for the Adafruit-Itsybitsy-M4
+
+### General information
+
+![Adafruit-Itsybitsy-M4 compared to a quarter dollar](https://cdn-learn.adafruit.com/assets/assets/000/055/465/large1024/adafruit_products_3800_quarter_ORIG_2018_06.jpg?1529192175)
+
+This is a small formfactor (only 1.4" long by 0.7" wide) SAM D51 board made by Adafruit.
+
+The board features one red LED (LD1), one DotStar / APA102 RGB LED, a reset button as well as
+21 configurable external pins(6 of which can be analog in).
+
+### Links
+
+- [Overview](https://learn.adafruit.com/introducing-adafruit-itsybitsy-m4?view=all)
+- [Schematics](https://learn.adafruit.com/assets/55481)
+
+### Flash the board
+
+The board is flashed using its on-board [boot loader](https://github.com/adafruit/Adafruit_nRF52_Bootloader).
```

Comment:
This page features a nrf52 bootloader. Shouldn't it be https://github.com/adafruit/uf2-samdx1 instead ?